goa/LOCALIZE.md

```diff
@@ -1,41 +1,73 @@
+Last updated: August 8, 2019
+
 ## Prerequisites
 Install Cocos Studio for [Mac](http://cocos2d-x.org/filedown/CocosForMac-v3.10.dmg) or [Windows](http://cocos2d-x.org/filedown/CocosForWin-v3.10.exe)
 
 This is used for creating scene assets representing the monster characters used in the game
 
-## Creation of new language support 
-- Create a new folder under Resources/res/lang/. Lets say the folder name is xyz
+## Steps to localize only instructions
+ 	
+* To build application with a new locale, add a new productFlavor in [build.gradle](https://github.com/maqsoftware/Simple-Class/blob/master/goa/proj.android-studio/goa/build.gradle) file.
+* Localize all the tutorial videos that are present in [help](https://github.com/maqsoftware/Simple-Class/tree/master/goa/Resources/res/lang/eng/help) section.
+* The [lang](https://github.com/maqsoftware/Simple-Class/tree/master/goa/Resources/res/lang/eng) folder contains a ***eng_en.po*** file, which contains strings in the `msgstr` tags, should also be localized and the new file should be saved as ***eng_[lang].po***.
+
+  ```	
+  msgid "text"	
+  msgstr "text in local language"
+  ```
+  After editing the ***.po*** file, a ***.mo*** file should be generated using an external utility ([sample](https://po2mo.net/)).
+* The [config](https://github.com/maqsoftware/Simple-Class/tree/master/goa/Resources/res/config) folder contains JSON files that can be localized.
+ 	* Update the `title` in _alphabet_game_map_en.json_, _shapes_game_map_en.json_, _words_game_map_en.json_, and _writing_game_map_en.json_
+	* The help instruction is contained in the ***game_levels_en.json*** file with `help` as the key for each section.	
+      ```json	
+      "miningbg": [	
+          {	
+            "levels": [	
+              1	
+            ],	
+            "help": "help/instruction text in local language",	
+            "video": "help.webm"	
+          }	
+        ]	
+      ```
+    * Save all the JSON files in [config](https://github.com/maqsoftware/Simple-Class/tree/master/goa/Resources/res/config) folder and append the ***lang*** code in their names. For example _grammar_game_map_sw.json_.
+* Add a new `localeCode` in [LevelHelpScene.h](https://github.com/maqsoftware/Simple-Class/blob/master/goa/Classes/menu/LevelHelpScene.h) file as per the build variant.
```

Comment:
Make it "Update the locale code" instead of "Add a new locale code" as per the **desired** build variant